projects/1.18/assets/skyopolis-mod/skyopolis/lang/zh_cn.json

```
@@ -0,0 +1,39 @@
+{
+  "item.skyopolis.leafy_string": "叶状线",
+  "item.skyopolis.organic_seeds": "有机种子",
+  "item.skyopolis.wooden_shears": "木剪刀",
+  "item.skyopolis.andesite_pebble": "安山岩碎块",
+  "item.skyopolis.granite_pebble": "花岗岩碎块 ",
+  "item.skyopolis.diorite_pebble": "闪长岩碎块",
+  "item.skyopolis.stone_pebble": "石头碎块",
+  "item.skyopolis.gravel_dust": "沙砾粉",
+  "item.skyopolis.sand_dust": "沙子粉",
+  "item.skyopolis.grout_dust": "水泥粉",
```

同上，`dust` 是 `尘土`，而且 `砖泥` 翻译错了，这一看都没开游戏
```suggestion
  "item.skyopolis.gravel_dust": "沙砾尘土",
  "item.skyopolis.sand_dust": "沙子尘土",
  "item.skyopolis.grout_dust": "砖泥尘土",
```